packages/medusa-payment-paypal/src/core/__mocks__/paypal-sdk.ts

```diff
@@ -0,0 +1,88 @@
+import { PaymentIntentDataByStatus } from "../../__fixtures__/data"
+
+export const SUCCESS_INTENT = "right@test.fr"
+export const FAIL_INTENT_ID = "unknown"
+export const INVOICE_ID = "invoice_id"
+
+export const PayPalMock = {
+  cancelAuthorizedPayment: jest.fn().mockImplementation(() => {
+    return {
+      status: "VOIDED"
+    }
+  }),
+  captureAuthorizedPayment: jest.fn().mockImplementation((id) => {
+    if (id === FAIL_INTENT_ID) {
+      throw new Error("Error.")
+    }
+
+    return {
+      id: "test",
+      capture: true,
+    }
+  }),
+  refundPayment: jest.fn().mockImplementation((paymentId) => {
+    if (paymentId === FAIL_INTENT_ID) {
+      throw new Error("Error")
+    }
+
+    return undefined
+/*
+    return {
```

Comment:
q: what are these commented out bits for?

Reply:
nice catch 💪